### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by Ashwani-Verma-07**
Make new changes to fonts and make points for huge sentence for reading easily.

### CODE REVIEW

**Review by Vimux — CHANGES_REQUESTED** [review]

File: exampleSite/content/docs/faq.md [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
-sidebar: false
-pager: false
-weight: 3
-menu:
+Date: 2022-01-24T14:00:00.000Z
+Authorbox: False
+Sidebar: False
+Pager: False
+Weight: 3
+Menu:
```
[the remark this answers is not in the document]
Comment (Ashwani-Verma-07) [reply exampleSite/content/docs/faq.md:10]: ok

File: exampleSite/content/docs/faq.md [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
   main:
     name: FAQ
 ---
 
-Browse this FAQ page to find a collection of answers to frequently asked questions that have not been covered elsewhere
+Browse this FAQ page to find a new collection of answers to frequently asked questions that have not been covered elsewhere
```
[the remark this answers is not in the document]
Comment (Ashwani-Verma-07) [reply exampleSite/content/docs/faq.md:15]: ok

**Comment by Ashwani-Verma-07** [comment]
now all errors done by me resolved and add new words

**Review by Vimux — CHANGES_REQUESTED** [review]
Good, but the problem with "answers" list is still the same.

**Comment by Ashwani-Verma-07** [comment]
done

File: exampleSite/content/docs/faq.md [hunk 3]
```diff
@@ -17,12 +17,17 @@ in the documentation.
...
 <!--more-->
 
-The answers have been categorized into two groups. Those are answers to general questions without any lines of code, and
-those are answers to technical questions with code snippets, step-by-step instructions, etc.
+The answers have been categorized into two groups:
+
+
+1. Answers to general questions without any lines of code.
+
+
```
Comment (Vimux) [inline exampleSite/content/docs/faq.md:25]: Extra empty lines

File: exampleSite/content/docs/faq.md [hunk 4]
```diff
@@ -17,12 +17,17 @@ in the documentation.
 
 <!--more-->
 
-The answers have been categorized into two groups. Those are answers to general questions without any lines of code, and
-those are answers to technical questions with code snippets, step-by-step instructions, etc.
+The answers have been categorized into two groups:
+
```
Comment (Vimux) [inline exampleSite/content/docs/faq.md:21]: Extra empty line

File: exampleSite/content/docs/faq.md (hunk 3 above)
Comment (Ashwani-Verma-07) [reply exampleSite/content/docs/faq.md:25]: yes this is for new line and give gap between the two
if there if error in hosted link then let me know

File: exampleSite/content/docs/faq.md (hunk 4 above)
Comment (Ashwani-Verma-07) [reply exampleSite/content/docs/faq.md:21]: yes this is for new line and give gap between the two
if there if error in hosted link then let me know

**Comment by Vimux** [comment]
Fix this, please.

**Review by Vimux — APPROVED** [review]

**Pull request closed without merging**